src/components/baseComponents/comments/CommentItem.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Grid, Box, SvgIcon, IconButton, Collapse } from '@material-ui/core';
+import { green, red } from '@material-ui/core/colors';
+import { withStyles, createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+
+import sprite from "assets/img/icons-sprite.svg";
+
+import { CommentChildProps } from './CommentChild';
+
+import moment from 'moment';
+import ReplyCommentPanel from './ReplyCommentPanel';
+import { Comment } from 'model/comments';
+
+interface CommentItemProps {
+  comment: Comment;
+}
+
+const Stretch = withStyles({
+	root: {
+		flexGrow: 1
+	}
+})(Grid);
+
+const GreenRedTheme = createMuiTheme({
+	palette: {
+		primary: green,
+		secondary: red
+	}
+});
+
+const CommentItem: React.FC<CommentItemProps> = props => {
+  const [replyPanelShown, setReplyPanelShown] = React.useState(false);
+
+	return (
+	<Grid item>
+		<Box marginX={2} marginY={2} padding={2} borderRadius={15} bgcolor="#ffffff">
+			<Grid container direction="column">
+				<Grid item container direction="row">
+					<Stretch item>
+            <h4><b>{props.comment.author.firstName} {props.comment.author.lastName}</b></h4>
+					</Stretch>
+					<ThemeProvider theme={GreenRedTheme}>
```

Comment:
@william-gooch this will be hard for Farman to style. try to add className instead